code/modules/organs/external/head.dm

```diff
@@ -122,6 +122,9 @@
 
 /obj/item/organ/external/head/proc/get_hair_icon()
 	var/image/res = image(bodytype.icon_template,"")
+	if(!owner)
```

Comment:
Won't that mean decapitated heads are going to turn bald once they get dumped?

Reply:
Yep. Heads need reworking to track the overlays directly instead of relying on owner.